pkg/csi/service/common/commonco/k8sorchestrator/k8sorchestrator.go

```diff
@@ -260,7 +343,7 @@ func configMapAdded(obj interface{}) {
 
 	fssConfigMap, ok := obj.(*v1.ConfigMap)
 	if fssConfigMap == nil || !ok {
-		log.Warnf("configMapAdded: unrecognized object %+v", obj)
+		log.Warnf("configMap add/update event: unrecognized object %+v", obj)
```

Comment:
This will create confusion while debugging when we want to know when config-map was added and when config-map was updated.

Reply:
Yes changed it